transforms/__testfixtures__/class-flow1.output.js

```diff
@@ -0,0 +1,59 @@
+/* @flow */
+
+var React = require('react');
+
+class Component extends React.Component {
+  props: {
+    optionalArray?: Array<any>,
+    optionalBool?: boolean,
+    optionalFunc?: Function,
+    optionalNumber?: number,
+    optionalObject?: Object,
+    optionalString?: string,
+    optionalNode?: any,
+    optionalElement?: any,
+    optionalMessage?: Message,
+    optionalEnum?: 'News' | 'Photos' | 1 | true | null,
+    optionalUnion?: string | number | Message,
+    optionalArrayOf?: Array<number>,
+    optionalObjectOf?: {[key: string]: number,},
```

Comment:
Will remove the trailing comma later: see https://github.com/benjamn/recast/pull/291#issuecomment-227965859